src/tests/ftest/util/nvme_utils.py

```diff
@@ -177,6 +178,12 @@ def start_ior_thread(self, create_cont, operation):
         if 'Auto_Read' in operation or operation == "Read":
             create_cont = False
             self.ior_local_cmd.flags.value = self.ior_read_flags
+            if 'Perf' in operation:
+                self.ior_local_cmd.block_size.update(
+                    self.ior_local_cmd.block_size.value * 10, 'ior.block_size')
```

Comment:
This code needs to be executed even if this isn't a read opperation - for `Perf_Auto_Write`:
```suggestion

        if 'Perf' in operation:
            self.ior_local_cmd.block_size.update(
                self.ior_local_cmd.block_size.value * 10, 'ior.block_size')
```